subsys/bluetooth/controller/Kconfig

```diff
@@ -86,4 +86,39 @@ choice SOFTDEVICE_CONTROLLER_VARIANT
 
 endchoice
 
+if BT_CTLR_ECDH
+choice BT_CTLR_ECDH_LIB
+	prompt "SoftDevice Controller ECDH library"
+
+config BT_CTLR_ECDH_LIB_OBERON
+	select NRF_OBERON
+	bool "Use nrf_oberon SW crypto library for ECDH"
+
+config BT_CTLR_ECDH_LIB_TINYCRYPT
+	select TINYCRYPT
+	select TINYCRYPT_ECC_DH
+	bool "Use TinyCrypt library library for ECDH"
+
+endchoice
+
+config BT_CTLR_ECDH_STACK_SIZE
+	# NOTE: This value is derived from other symbols and should only be
+	# changed if required by architecture
+	int "Bluetooth controller ECDH thread stack size"
+	default 800 if BT_CTLR_ECDH_LIB_OBERON
+	default 1024 if BT_CTLR_ECDH_LIB_TINYCRYPT
+	help
+	  NOTE: This is an advanced setting and should not be changed unless
+	  absolutely necessary
```

Comment:
Warning, if user changes from `BT_CTLR_ECDH_LIB_OBERON` to `BT_CTLR_ECDH_LIB_TINYCRYPT` using menuconfig (or the other  way around), then this value will not be updated.

So only informing the user not to change this value is not sufficient, due to the stuck symbol problem if updating the choice.
https://docs.zephyrproject.org/latest/guides/kconfig/tips.html#stuck-symbols-in-menuconfig-and-guiconfig

Reply:
I see the prompt have been removed, which effectively addresses my concerns.